src/components/WeatherApp.jsx

```diff
@@ -1,56 +1,110 @@
-import React, { useEffect } from 'react'
+import React, { useEffect, useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faDroplet } from '@fortawesome/free-solid-svg-icons'
+import BeatLoader from "react-spinners/BeatLoader";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 import { FiSearch } from 'react-icons/fi'
-import { BsDropletFill } from 'react-icons/bs'
 import { FaWind } from 'react-icons/fa'
 const WeatherApp = () => {
+    const [weatherData, setWeatherData] = useState({})
+    const [isLoading, setIsLoading] = useState(false)
+    const [inputName, setInputName] = useState("Pune");
+    const [saveInputText, setSaveInputText] = useState("Pune");
+    const [error, setError] = useState({ errorText: "", showError: "removeErrorSymbol" })
```

Comment:
Why do we need this default error state, what does it do?

Reply:
yes I understood. No need of error state.